letsencrypt/client.py

```diff
@@ -331,25 +340,14 @@ def deploy_certificate(self, domains, privkey_path,
             self.installer.save("Deployed Let's Encrypt Certificate")
 
         # sites may have been enabled / final cleanup
-        with error_handler.ErrorHandler(self._rollback_and_restart):
+        restart_error_handler = error_handler.ErrorHandler(functools.partial(
+            self._rollback_and_restart,
+            "We were unable to install your certificate, however, we "
```

Comment:
Could we make a class variable of this message as well, like _SUCCESSFUL_REDIRECT_RECOVERY_MSG. That'd be a consistent way of managing the different return messages.